### DISCUSSION TITLE
最新full-refractor分支构建的安卓版未进行应用签名

### DISCUSSION THREAD

**Discussion by JingLuoY** [amll-dev/applemusic-like-lyrics#256] [Q&A] [answered]
![Uploading 1726996894689_d.jpg…]()
安卓在没有核心破解下手机无法安装

**Comment by Steve-xmh** [comment, marked as answer]
请借助自签名工具或者adb以实现安装功能
开发构建暂不考虑对应用签名

**Reply by JingLuoY** [reply]
好的，现在安卓端无法导入本地歌曲，而且界面仍保留Windows的最大化，最小化以及关闭按钮
